docs/development.md

```diff
@@ -52,13 +54,76 @@ Lastly, spin up the box:
 vagrant up centos7-devel
 ```
 
-The box can now be accessed via ssh and the Rails server started directly (this assumes you are connecting as the default `vagrant` user):
+If you get an error about the host CPU not providing the 'svm' feature but you most assuredly do have an Intel CPU and virtualization is most definitely turned on, add this to your Vagrantfile.
 
-    vagrant ssh <deployment>
-    cd /home/vagrant/foreman
-    sudo service iptables stop
-    bin/rails s -b 0.0.0.0
+```
+Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |domain|
+    domain.cpu_mode = "host-passthrough"
+  end
+end
+```
+
+Try to bring it up again with `vagrant up centos7-devel` after making that change.
+
+If you get an error related to your user not having privileges to execute libvirt commands, create a polkit rule similar to the following:
```

Comment:
👏

Reply:
On Fedora 27 it's sufficient to be in the libvirt group since they ship the policy by default.